.nvmrc

```diff
@@ -0,0 +1 @@
+v16.15.1
```

Comment:
They typically are. Right now, [Node LTS](https://nodejs.org/) is actually 18.14.2.

Reply:
Yea, good call. I just changed `.nvmrc` to simply accept any minor or patched version of `v16`. I also removed the `engines: {}` definition in package.json because it's really not needed anymore and throws a loud warning right before `nvm` switches Node versions and resolves the issue :)